tests/ceph_installer/test_ansible.py

```diff
@@ -33,6 +33,26 @@ def run(ceph_cluster, **kw):
     ceph_cluster.custom_config_file = test_data.get("custom-config-file")
     cluster_name = config.get("ansi_config").get("cluster")
 
+    # For cdn container installation GAed container parameters
+    # needs to override as below,
+    #
+    # config:
+    #     use_cdn: True
+    #     ansi_config:
+    #       ceph_repository_type: cdn
+    #       ceph_docker_image: "rhceph/rhceph-4-rhel8"
+    #       ceph_docker_image_tag: "latest"
+    #       ceph_docker_registry: "registry.redhat.io"
+
+    # Enables cdn ansible config when --build set to relesed
+    build_type = config.get("build_type", None)
+    if build_type == "released":
+        config.get("ansi_config")["use_cdn"] = True
+        config.get("ansi_config")["ceph_repository_type"] = "cdn"
+        config.get("ansi_config")["ceph_docker_image"] = "rhceph/rhceph-4-rhel8"
+        config.get("ansi_config")["ceph_docker_image_tag"] = "latest"
+        config.get("ansi_config")["ceph_docker_registry"] = "registry.redhat.io"
+
```

Comment:
We need to change this to
```
config["ansi_config"]["use_cdn"] = True
config["ansi_config"]["ceph_repository_type"] = "cdn"
config["ansi_config"]["ceph_docker_image"] = "rhceph/rhceph-4-rhel8"
config["ansi_config"]["ceph_docker_image_tag"] = "latest"
config["ansi_config"]["ceph_docker_registry"] = "registry.redhat.io"
```

Reply:
IMO, `use_cdn` was enabling all right repos in existing test cases. we may just need to enable cdn in case of `--build released`.

https://github.com/red-hat-storage/cephci/blob/master/tests/ceph_installer/test_ansible.py#L47-L64